### PR TITLE
EKS 1.15 Support

### PR DESCRIPTION
**Pull request by sc250024**
# PR o'clock

## Description

Upgrading module to support EKS 1.15. Resolves issue https://github.com/terraform-aws-modules/terraform-aws-eks/issues/775.

### Checklist

- [X] Change added to CHANGELOG.md. All changes must be added and breaking changes and highlighted
- [x] CI tests are passing
- [X] README.md has been updated after any changes to variables and outputs. See https://github.com/terraform-aws-modules/terraform-aws-eks/#doc-generation

### CODE REVIEW

**Comment by sc250024** [comment]
@max-rocket-internet I've added you as a collaborator on the fork repository here if you need to make changes: https://github.com/sc250024/terraform-aws-eks

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ project adheres to [Semantic Versioning](http://semver.org/).
 - Add `default_cooldown` and `health_check_grace_period` options to workers ASG (by @ArieLevs)
 - Fix support for ASG max instance lifetime for workers (by @barryib)
 - Add support for envelope encryption of Secrets (by @babilen5)
+- Added support for EKS 1.15 (by @sc250024)
```
Comment (max-rocket-internet) [inline CHANGELOG.md]: Please mark it as `breaking`

Comment (sc250024) [reply CHANGELOG.md]: Yep, no problem!

Comment (sc250024) [reply CHANGELOG.md]: Done here => https://github.com/terraform-aws-modules/terraform-aws-eks/pull/776/commits/a1c5510c2193b5f53302a45b4c759a9c5dc53bfb

**Comment by abdennour** [comment]
Does tag `"kubernetes.io/cluster/${local.cluster_name}"` not compatible with EKS 1.15 ? 
I am seeing a lot of removals of tags.

**Comment by sc250024** [comment]
@abdennour Check out https://github.com/terraform-aws-modules/terraform-aws-eks/issues/775. The VPC tagging requirement is no longer needed for clusters with Kubernetes 1.15 and above. I removed those VPC tags from the examples since starting with v10.0.0 of this module they're no longer needed.

**Pull request merged**